server/src/main/java/io/crate/metadata/IndexReference.java

```diff
@@ -210,6 +232,11 @@ public Map<String, Object> toMapping() {
             mapping.put("analyzer", analyzer);
             mapping.put("type", "text");
         }
+
+        if (columns.isEmpty() == false) {
+            mapping.put("sources", columns.stream().map(ref -> ref.column().fqn()).collect(Collectors.toList()));
+        }
```

Comment:
copy_to/sources is `CREATE TABLE` specific thing which is currently handled by `toMapping` and has only `copy_to`.
Even after merging this PR we won't have `sources` since it's not used on user facing code path, it's just preparation for the next PR.

Usage in the next PR would smth like use `sources` to resolve real References and add them into new `CreateTableRequest`. We agreed to stream not names but references which is fine for DDL.

In the CREATE TABLE refactoring we agreed to get rid of toMapping (which I already did) and we then finally have sources (this code will will be finally used) and copy_to will be gone together with `AnalyzedColunDefinition.toMapping`